src/containers/balance/index.js

```diff
@@ -1,31 +1,71 @@
 import React, { PureComponent } from 'react'
```

Comment:
what is this balance page for? is it just an example container?

Reply:
Yeah, I needed an example container so I could write an integration test with the whole redux saga fetch/loading/{receive,failed} flow.